mount-daemon/controllers/clientmount_controller.go

```diff
@@ -268,6 +268,13 @@ func (r *ClientMountReconciler) mount(ctx context.Context, clientMountInfo dwsv1
 		return err
 	}
 
+	if clientMountInfo.SetPermissions {
```

Comment:
Couldn't you check if either user ID or group ID is non-zero?

Reply:
That would prevent you from setting the user/group to root. This Is part of DWS, so it's supposed to be more general purpose than what we need for Rabbit. For Rabbit what you described would work fine since we expect the mounts to always be for a real user.